### ISSUE TITLE
Incorrect station display

### ISSUE THREAD

**Issue by jamesbascle** [tautomer/EDMasscacreMissionCompanion#3]
* MassacreCompanion version: 0.1.0
* Python version: 3.8.7
* Operating System: Windows 10

### Description

View the correct station the mission came from.

### What I Did

Accepted a bunch of missions in Goplang, HIP 108729, and HIP 109787 targeting HIP 108701.  The tool shows that all the missions' station is in MacCready Dock and Kagawa Ring in 108701, instead of the station the mission came from in the source systems.

Thanks for this tool, it's very useful!

**Comment by tautomer** [comment]
Hmmm... I never went to multiple source systems for missions, so I never got a chance to test the behavior like this.

I'm not very sure why this would happen. The system & station names should go with whatever from the log file.
Could you please send me the ED log files which contain the info on accepting these missions, so that I can take a look? You can use Google Drive or a pastebin like [Ubuntu Pastebin](https://paste.ubuntu.com/).

One thing though... I'm quite busy recently with my research and job searching, so I can't guarantee a timing for the fix. In fact, I haven't played ED at all this weekend. 😂

**Comment by jamesbascle** [comment]
Sure, here are the massacre entries from my latest log.

https://paste.ubuntu.com/p/3BsPzz9cQM/

What seems to be happening is that the game is saying the destination station is the station nearest the signal sources in the target system...it may just be a limitation of the journal files.

And no problem of course, you're under no obligation to fix any of this in any time frame.  The software is greatly helpful for gathering and organizing massacre missions as is and I'm very thankful for your work on it!

**Comment by tautomer** [comment]
I think it's actually an intended behavior. One thing, as you said, it goes with the journal.

The other thing is that the "destination system" is more like the "current destination system". Say, before you finish the mission, the "destination" is indeed HIP 108701 as you will go there to kill pirates, and when you are done, the game redirect this value back to the source systems for the rewards. In this sense, the destination part should be what it's like now. What's your opinion on this?
(If someone is crazy enough to do massacre missions targeting multiple systems, it will be important to know which target system one should go for each mission. It's unlikely to happen, but demonstrates how the current way will help.) 

The destination station part is indeed nothing more than a placeholder before it's redirected back to the source station. I doubt if people will chase down the pirates following the signals, but if they do, the station name could also be helpful.

**Comment by jamesbascle** [comment]
That makes sense.  If it's possible to actually have it list the station the mission came from in addition, that could be a helpful since it's nice to just have the least amount of stations to have to turn in at, ideally 1 in each system.  It's not a critical feature but if it's possible I think it would be useful.

**Comment by tautomer** [comment]
I agree with you on this. Then I need to take care of the docking/undocking events as well, since this is probably the only way to know which station it is. Let's see when I can get some free time!

**Comment by Xendi** [comment]
> That makes sense. If it's possible to actually have it list the station the mission came from in addition, that could be a helpful since it's nice to just have the least amount of stations to have to turn in at, ideally 1 in each system. It's not a critical feature but if it's possible I think it would be useful.

I think if the title bar said "Next Waypoint" or similar instead of "Station" it might be more intuitive.

Thanks also, from me for this app it's great!
